packages/website/pages/index.tsx

```diff
@@ -49,6 +49,16 @@ class HomePage extends React.Component {
     }
   }, 100);
 
+  getReadmePanelProps = () => {
+    return {
+      IconComponent: MediaDocIcon,
+      label: 'Get Started',
+      color: colors.R400,
+      description: 'Everything you need to get up and running',
+      imgSrc: '/static/code.png',
```

Comment:
We'll probably want to use a different image but it's fine if we address this in a future PR